ampersand-multifield-view.js

```diff
@@ -73,9 +98,25 @@ var MultiFieldView = View.extend({
   },
 
   isValid: function() {
-    return this.fields.every(function(field) {
+    var fieldsValid =  this.fields.every(function(field) {
       return field.valid;
     });
+
+    var valid = !this.runTests();
+
+    return fieldsValid && valid;
```

Comment:
you can skip the intermediary and make this a bit more efficient; if a field isn't valid, no need to check if the multi-field is valid:
```js
return fieldsValid && !this.runTests();
```
I wish I could think of a better name for `runTests`, but alas...